pkg/geo/geomfn/azimuth.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package geomfn
+
+import (
+	"math"
+
+	"github.com/twpayne/go-geom"
+)
+
+// Azimuth returns the azimuth in radians of the segment defined by the given point geometries.
+// The azimuth is angle is referenced from north, and is positive clockwise.
+// North = 0; East = π/2; South = π; West = 3π/2.
+func Azimuth(a *geom.Point, b *geom.Point) *float64 {
```

Comment:
actually, do you mind moving the logic in geo_builtins such that Azimuth only takes in `(a *geo.Geometry, b *geo.Geometry)`, to be consistent with the rest of the function signatures in ehre?

Reply:
Done.